izpack-gui/src/main/java/com/izforge/izpack/gui/IzPanelLayout.java

```diff
@@ -106,7 +106,6 @@
         catch (ClassNotFoundException e)
         {
             pathSelectionPanel = null;
-            e.printStackTrace();
```

Comment:
For better analysis the catched exception still should be logged in maybe fine mode using the 
`java.util.logging.Logger#log(Level level, String msg, Throwable thrown)` method.

Reply:
Done.